docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cmake \
     ../src
 RUN make VERBOSE=1 -j2
 RUN ccache -s
-RUN ctest -E "walker:SkewNormalPDF_pe8"
```

Comment:
I really like this solution to exclude tests only for travis, instead of labeling them in any way. This way the exclusion only affects travis and does not exlude tests in any other venue. Awesome!